model/lifecycle.go

```diff
@@ -767,6 +771,19 @@ func CreateTasksFromGroup(in BuildVariantTaskUnit, proj *Project, requester stri
 			Activate:         in.Activate,
 			CommitQueueMerge: in.CommitQueueMerge,
 		}
+		// Overwrite limiting when tasks will run from project task
```

Comment:
We're not overwriting, we're defaulting right?

Reply:
Maybe what we should do instead is always call bvt.Populate (line 788) but then only add to tasks (line 789) if 787 is met. This would prevent us from special casing ahead of time.

Reply:
done